sdk/tests/android/testapp/src/androidTest/java/software/amazon/awssdk/iottest/MainActivityTest.kt

```diff
@@ -0,0 +1,152 @@
+package software.amazon.awssdk.iottest
```

Comment:
A missing copyright header? Not sure about this tbh

Reply:
Adding them everywhere =P